src/interrupt.c

```diff
@@ -472,11 +473,84 @@ void unregister_SP_handler( void (*callback)() )
     __unregister_callback(&SP_callback,callback);
 }
 
+
+/**
+ * @brief Register a timer callback
+ * 
+ * The callback will be used when the timer interrupt is triggered by the CPU.
+ * This happens when the COP0 COUNT register reaches the same value of the
+ * COP0 COMPARE register.
+ * 
+ * This function is useful only if you want to do your own low level programming
+ * of the internal CPU timer and handle the interrupt yourself. In this case,
```

Comment:
Is it possible to disable the mechanism we have at inthandler.S?

Reply:
Which one?

Reply:
This was a comment I forgot to delete. I meant how the users would not use the library version but instead implement their own. Then I realized, they just don't use the library timers...